bin/generate-mailing-list-us-only.rb

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env ruby
+require_relative '../pegasus/src/env'
+require 'cdo/solr'
+require src_dir 'database'
+
+SETS = {
```

Comment:
can probably delete all this SETS stuff and other stuff left over from the original